mediator/med_io_mod.F90

```diff
@@ -457,7 +457,7 @@ subroutine med_io_write_FB(filename, iam, FB, whead, wdata, nx, ny, nt, &
     real(r8), pointer             :: fldptr1(:)
     real(r8), pointer             :: fldptr2(:,:)
     real(r8), allocatable         :: ownedElemCoords(:), ownedElemCoords_x(:), ownedElemCoords_y(:)
-    character(len=number_strlen)  :: cnumber
+    character(16)                 :: cnumber
```

Comment:
This should use a parameter and not a local value.

Reply:
Good point - I just fixed this and pushed it back.